app/controllers/categories_controller.rb

```diff
@@ -1,6 +1,16 @@
 # frozen_string_literal: true
 
 class CategoriesController < ApplicationController
+  def show
+    category = Category.find(params[:id])
+    render json: CategoryPresenter.present(category)
+  end
+
+  def subcategories
+    categories = Category.where("id in (select child_id from category_relationships where parent_id=?)", params[:id])
```

Comment:
I think this just searches one sublevel. I feel that I saw in the past that we had like a 3 or 4-level deep list of subcategories, but is that not the case today or not necessary yet?